jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/controller/CodeReviewCommentController.java

```diff
@@ -86,11 +86,28 @@ public void initBinder(WebDataBinder binder) {
     @RequestMapping(value = "/reviews/{postId}/json", method = RequestMethod.GET)
```

Comment:
Can we rely on `Accepts` header instead of adding `/json` into the path? And default to `application/json` in case the header is empty. 

Another way we usually change this in Spring MVC is by parameter `?format=json` or by extension `.json`, but in any case we can default to JSON.